frontend/www/js/omegaup/components/login/Login.vue

```diff
@@ -85,5 +85,26 @@ export default class Login extends Vue {
   usernameOrEmail: string = '';
   password: string = '';
   T = T;
+  window = window;
+
+  mounted() {
+    window.gapi.signin2.render('google-signin', {
+      scope: 'profile',
+      width: 45,
+      height: 45,
+      longtitle: false,
+      theme: 'light',
+      onsuccess: this.onHandleSuccess,
+      onfailure: this.onHandleFailure,
+    });
+  }
+
+  onHandleSuccess(googleUser: any) {
```

Comment:
nit:

```suggestion
  onSuccess(googleUser: gapi.auth2.GoogleUser) {
```

una función "onFoo" significa que está manejando un evento llamado "foo". en este caso suena a que esto está manejando un evento llamado "handle success": eso sería equivalente a un evento que se manda llamar _después_ de terminar de manejar el evento que se manda llamar al hacer login exitosamente.

igual abajo.

Reply:
Listo. Ya quedaron los nombres de las funciones.

Con respecto al tipo del argumento, el linter se está quejando y no me permite hacer push:
```
frontend/www/js/omegaup/components/login/Login.vue:101:25: [eslint] 'gapi' is not defined. [no-undef]
      onSuccess(googleUser: gapi.auth2.GoogleUser) {
                            ^^^^
```

he intentado arreglarlo de varias formas: 
```
import 'gapi';
```
o:
```
import gapi from 'gapi';
```
pero ninguna funcionó. Desconozco como arreglarlo.

Reply:
esto funciona? https://stackoverflow.com/a/43751465/688337

Reply:
Deja intentar

Reply:
Si, ya con eso deja de quejarse `ESLint`, gracias!!